ttnn/cpp/ttnn/operations/ccl/ccl_op_fusion.cpp

```diff
@@ -159,7 +159,7 @@ void MatmulFusedOpSignaler::push_matmul_fused_op_rt_args(
         out_rt_args.push_back(static_cast<uint32_t>(this->output_page_offset));
         out_rt_args.push_back(static_cast<uint32_t>((this->ring_size - 1) * this->output_page_offset));
     }
-    out_rt_args.push_back(static_cast<uint32_t>(this->is_clockwise_dir));
+    out_rt_args.push_back(static_cast<uint32_t>(this->is_clockwise_dir ? Direction::CCW : Direction::CW)); // Counter-clockwise direction is the first since it has local slice
```

Comment:
Please elaborate in the comment on why starting with CCW and going to CW is considered clockwise and starting with CW and going to CCW is considered counterclockwise (not clockwise).